lld/COFF/Writer.cpp

```diff
@@ -2310,6 +2313,18 @@ void Writer::createECChunks() {
       ctx.symtab.findUnderscore("__arm64x_redirection_metadata");
   replaceSymbol<DefinedSynthetic>(entryPointsSym, entryPointsSym->getName(),
                                   entryPoints);
+
+  for (auto thunk : ctx.symtab.sameAddressThunks) {
+    wowthkSec->insertChunkAtStart(thunk);
```

Comment:
Does this mean that for each thunk, we insert it at the start, i.e. adding them in reverse order? That seems both a bit odd and inefficient. Wouldn't `wowthkSec` be empty here and we could just append to it?

Reply:
`wowthkSec` is also used by thunks provided by object files, so they would be inserted after those. I did it this way because it more closely matches MSVC's behavior (in terms of placement, not necessarily order). I moved the insertion into `createSection`, where `wowthkSec` is still empty. We also add some other chunks there.